vendor/bat-native-ads/src/bat/ads/internal/account/account.cc

```diff
@@ -329,8 +329,13 @@ void Account::OnDidRefillUnblindedTokens() {
 void Account::OnCaptchaRequiredToRefillUnblindedTokens(
     const std::string& captcha_id) {
   const WalletInfo& wallet = GetWallet();
-  AdsClientHelper::GetInstance()->ShowScheduledCaptchaNotification(wallet.id,
-                                                                   captcha_id);
+#if BUILDFLAG(IS_ANDROID) || BUILDFLAG(IS_IOS)
+  AdsClientHelper::GetInstance()->ShowScheduledCaptchaNotification(
+      wallet.id, captcha_id, false);
+#else
+  AdsClientHelper::GetInstance()->ShowScheduledCaptchaNotification(
+      wallet.id, captcha_id, true);
+#endif  // BUILDFLAG(IS_ANDROID) || BUILDFLAG(IS_IOS)
```

Comment:
Ideally, the ads lib should not have these conditions. Could we please decouple this logic to the captcha code? Unless I am missing something

Reply:
i tried to decouple code in previous commit but in that case we need to update code for desktop changes. So after checking with @emerick we decided to go with the change.

Reply:
`// !BUILDFLAG(IS_ANDROID) && !BUILDFLAG(IS_IOS)` comment for `#else`

Reply:
I am still thinking about this, but for now `false`/`true` need param comments, i.e. `/* should_show_notification */ false`

Reply:
also rather than duplicating `AdsClientHelper::GetInstance()->ShowScheduledCaptchaNotification(` we should set a  `bool`